src/app/app.component.html

```diff
@@ -6,40 +6,45 @@ <h2>Unsupported Browser</h2>
     </div>
   </div>
 }
-@if (currentUrlWithoutHash$ | async; as currentUrl) {
-  <div class="skip-to-content-link">
-    <!--
-    Can't use a simple relative link for the URL because we have base href set of /d/ set in non-dev environments
-    which is needed for links to assets but would break this.
-    -->
-    <a [href]="currentUrl.toString() + '#content'">Skip to content</a>
-  </div>
-}
 
-@if (isDataLoaded) {
-  <biggive-main-menu
-    [donateUrlPrefix]="donateUriPrefix"
-    [blogUrlPrefix]="blogUriPrefix"
-    [experienceUrlPrefix]="experienceUriPrefix"
-    [isLoggedIn]="isLoggedIn"
-    [smallCharityWeekEnabled]="true"
-    [someCampaignHasHomePageRedirect]="someCampaignHasHomePageRedirect()"
-  ></biggive-main-menu>
-}
+<div role="banner" aria-label="Menu &amp; skip">
+  @if (currentUrlWithoutHash$ | async; as currentUrl) {
+    <div class="skip-to-content-link">
+      <!--
+      Can't use a simple relative link for the URL because we have base href set of /d/ set in non-dev environments
+      which is needed for links to assets but would break this.
+      -->
+      <a [href]="currentUrl.toString() + '#content'">Skip to content</a>
+    </div>
+  }
+
+  @if (isDataLoaded) {
+    <biggive-main-menu
+      [donateUrlPrefix]="donateUriPrefix"
+      [blogUrlPrefix]="blogUriPrefix"
+      [experienceUrlPrefix]="experienceUriPrefix"
+      [isLoggedIn]="isLoggedIn"
+      [smallCharityWeekEnabled]="true"
+      [someCampaignHasHomePageRedirect]="someCampaignHasHomePageRedirect()"
+    ></biggive-main-menu>
+  }
+</div>
 
 <a id="content">
   <!-- target for "skip to content" link -->
 </a>
 <router-outlet></router-outlet>
 
-<biggive-footer
-  [headingLevel]="3"
-  [usePresetFooter]="true"
-  [donateUrlPrefix]="donateUriPrefix"
-  [blogUrlPrefix]="blogUriPrefix"
-  [experienceUrlPrefix]="experienceUriPrefix"
-  [smallCharityWeekEnabled]="true"
-></biggive-footer>
+<div role="contentinfo">
```

Comment:
could this be a <footer> element instead?

Also maybe something to do in components library instead of here so it can benefit the wordpress site at the same time?

Reply:
Yeah I think I'm inclined to leave this one for now so long as Axe seems happy with the updated version in other contexts like regression tests too. I agree it might make sense to give the inner `components` element a more general element and leave landmarks to the parent apps, if we work on it again.